src/pages/index.js

```diff
@@ -144,3 +190,72 @@ const cardUrlInput = addCardFormElement.querySelector("#add-url");
 // Adding a New Card //
 
 addNewCardButton.addEventListener("click", () => addCardWithPopupForm.open());
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "89dc4b2f-fab0-42f3-ad8c-2593f7f5189c",
+    "Content-Type": "application/json",
+  },
+});
+
+function handleLikeCard(card, cardId, isLiked) {
+  api
+    .updateLikeStatus(cardId, isLiked)
+    .then((updatedCard) => {
+      card.setIsLiked(updatedCard.isLiked);
+    })
+    .catch((err) => console.error(err));
+}
+
+function renderCardsAfterUserInfo() {
+  return Promise.all([api.getInitialCards(), api.getUserInfo()]).then(
+    ([cards, userData]) => {
+      cardSection.renderItems(cards);
+      userInfo.setUserInfo({
+        name: userData.name,
+        job: userData.about,
+      });
+      userInfo.setUserAvatar({
+        avatar: userData.avatar,
+      });
+    }
+  );
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**